src/main/java/com/hazelcast/cloud/Client.java

```diff
@@ -49,20 +45,22 @@ public static void main(String[] args) {
      * @param client - a {@link HazelcastInstance} client.
      */
     private static void mapExample(HazelcastInstance client) {
-        System.out.println("Now the map named 'map' will be filled with random entries.");
-
-        IMap<String, String> map = client.getMap("map");
-        Random random = new Random();
-        int iterationCounter = 0;
-        while (true) {
-            int randomKey = random.nextInt(100_000);
-            map.put("key-" + randomKey, "value-" + randomKey);
-            map.get("key-" + random.nextInt(100_000));
-            if (++iterationCounter == 10) {
-                iterationCounter = 0;
-                System.out.println("Current map size: " + map.size());
-            }
-        }
+        System.out.println("Now the map named 'cities' will be filled with some entries.");
```

Comment:
I don't think we need this line. We should avoid printing too much in the output.

Reply:
removed it